matrix/matrix.yaml

```diff
@@ -9,21 +9,5 @@
       "periodic": 5
       "do":
         "task": matrix.tasks.health
+        "gating": true
```

Comment:
Not that it matters overmuch, but I thought our discussion had been to put it at the rule level instead of the arg level?  TBH, I would be in favor of moving `task` up to the top level as well, doing away with `do` and replacing it with an explicit `args` or `task_args` field, to remove the unnecessary level of nesting.  At any rate, for something that is supported in some way across all tasks, I'm inclined to have it at the rule level.